tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -452,6 +452,22 @@ def test_interp_inf_to_nan(self, xp, dtype_y, dtype_x):
         fy = xp.asarray([0, 10], dtype=dtype_y)
         return xp.interp(x, fx, fy)
 
+    @testing.for_all_dtypes(name='dtype_2', no_bool=True, no_complex=True)
+    @testing.for_all_dtypes(name='dtype_1', no_bool=True, no_complex=True)
+    @testing.numpy_cupy_array_equal(type_check=False)
```

Comment:
Let's check return types.

Reply:
Fixed in 546002fada1ec43bfd58090ffb7afbda591c4731.